config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  root 'homes#index'
+  root 'beers#index'
   devise_for :users
 
+  resources :beers, only: [:index]
```

Comment:
if you have methods in your reviews controller, you need to update your routes accordingly. Would you need nested beer/review routes?